pages/index.vue

```diff
@@ -26,6 +29,32 @@ export default {
       { to: '/credit-card-generator', text: 'Credit Card Generator' }
     ]
   }),
+
+  computed: {
+    orderedLinks() {
+      const orderedLink = Array.from(this.links)
+
+      return orderedLink.sort((a, b) => this.compare(a, b))
+    }
+  },
+
+  methods: {
+    compare(a, b) {
+      const textA = a.text.toUpperCase()
+      const textB = b.text.toUpperCase()
+
+      let comparison = 0
+
+      if (textA > textB) {
+        comparison = 1
+      } else if (textA < textB) {
+        comparison = -1
+      }
+
+      return comparison
+    }
+  },
+
```

Comment:
Since `compare` function is not used anywhere, doesn't make sense to add in methods. I'd add that as a closure  inside the `orderedLinks` computed property itself. The following would be

```suggestion
  computed: {
    orderedLinks: vm => {
      const compare = (a, b) => {
        const textA = a.text.toUpperCase()
        const textB = b.text.toUpperCase()
        return textA === textB ? 0 : textA > textB ? 1 : -1
      }

      return vm.links.sort(compare)
    }
  },
```

Also, noticed that the same should apply to `default.vue` since it has almost the same items. If you could just do that, it would be great!